bokehjs/src/lib/models/tools/actions/custom_action.ts

```diff
@@ -0,0 +1,66 @@
+import {ActionTool, ActionToolView, ActionToolButtonView} from "./action_tool"
+import * as p from "core/properties"
+import {isFunction} from "core/util/types"
+
+export class CustomActionButtonView extends ActionToolButtonView {
+  model: CustomAction
+
+  css_classes(): string[] {
+    return super.css_classes().concat("bk-toolbar-button-custom-action")
```

Comment:
This is to provide a known target for Selenium test to be able to click this button.